### ISSUE TITLE
Total Equity not available in balance_sheet?

### ISSUE THREAD

**Issue by Rapid1898-code** [ranaroussi/yfinance#1043]
Hello - i would like to retrieve the
"Total Equity Gross Minority Interest"
from the balance-sheet eg. for the stock JPM

This should be 294,127,000 in 2021
![image](https://user-images.githubusercontent.com/57018730/181495858-06cae010-dee6-4448-8903-838a64849040.png)

But i only get this information when requesting the balance_sheet for JPM

```
                             2021-12-31    2020-12-31    2019-12-31    2018-12-31
Intangible Assets          6.376000e+09  4.180000e+09  5.518000e+09  6.878000e+09
Capital Surplus            8.841500e+10  8.839400e+10  8.852200e+10  8.916200e+10
Total Liab                 3.449440e+12  3.105403e+12  2.426049e+12  2.366017e+12
Total Stockholder Equity   2.592890e+11  2.492910e+11  2.343370e+11  2.304470e+11
Other Current Liab         1.145770e+11  9.955800e+10  7.556900e+10  1.030040e+11
Total Assets               3.743567e+12  3.384757e+12  2.687379e+12  2.622532e+12
Common Stock               4.105000e+09  4.105000e+09  4.105000e+09  4.105000e+09
Other Current Assets       2.773480e+11  2.356370e+11  2.322790e+11  2.074050e+11
Retained Earnings          2.722680e+11  2.369900e+11  2.232110e+11  1.992020e+11
Other Liab                 2.261000e+09  2.420000e+09           NaN           NaN
Good Will                  5.031500e+10  4.924800e+10  4.782300e+10  4.747100e+10
Treasury Stock            -1.054990e+11 -8.019800e+10 -8.150100e+10 -6.202200e+10
Other Assets               1.264438e+12  1.144340e+12  1.109292e+12  1.081167e+12
Cash                       7.212010e+11  5.032090e+11  2.171420e+11  2.380770e+11
Total Current Liabilities  3.151444e+12  2.795212e+12  2.108205e+12  2.063745e+12
Short Long Term Debt       2.277600e+10           NaN           NaN           NaN
Other Stockholder Equity  -8.400000e+07  7.986000e+09  1.548000e+09 -1.528000e+09
Property Plant Equipment   4.462300e+10  4.826400e+10  4.940000e+10  3.636200e+10
Total Current Assets       1.693889e+12  1.538408e+12  1.068584e+12  1.181001e+12
Long Term Investments      6.839260e+11  6.003170e+11  4.067620e+11  2.696530e+11
Net Tangible Assets        2.080920e+11  1.991390e+11  1.856950e+11  1.822280e+11
Short Term Investments     2.616310e+11  2.962840e+11  2.492230e+11  3.215790e+11
Long Term Debt             2.889790e+11  2.992630e+11  3.093390e+11  3.022720e+11
Inventory                  1.340000e+08  1.520000e+08  2.530000e+08  2.260000e+08
Accounts Payable           2.529897e+12  2.213223e+12  1.635858e+12  1.543482e+12
```

So i have only the "Common Stock Equity".
Is there any way to get the "Total Equity Gross Minority Intere"
I also saw that "Common Stock Equity" + "Preferred Stock Equity" will also sum up to the needed "Total Equity Gross Minority Intere" but i can also not find "Preferred Stock Equity" in the balance_sheet output.

Is there any way to get "Total Equity Gross Minority Intere"?

**Comment by ValueRaider** [comment]
Duplicate of https://github.com/ranaroussi/yfinance/issues/1004

Also, there is a pending PR that will fix this: https://github.com/ranaroussi/yfinance/pull/776

**Comment by ValueRaider** [comment]
@Rapid1898-code The probability of anything being merged in is near-zero, because owner is the only person who can merge. To solve, I'm thinking of setting up an "official" fork with multiple maintainers - do you have time spare to assist? It doesn't have to mean coding, can just mean reviewing PRs.

**Comment by Rapid1898-code** [comment]
Hello - yes this sound interesting - i would love to help.
KR, Max

**Comment by ValueRaider** [comment]
Correct financials are now available in pre-release version 0.2.0rc4.
